### PR TITLE
Updated steps to create classic sites.

### PR DESCRIPTION
**Pull request by derhallim**
M365 Admin Center will route users to the SP admin center. Existing steps are no longer valid. Fix for issue #2632

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/create-site-collection.md [hunk 1]
```diff
@@ -63,25 +63,27 @@ By using the new SharePoint admin center, you can create sites that use one of t
...
-    > If you have Office 365 Germany, sign in at https://portal.office.de. If you have Office 365 operated by 21Vianet (China), sign in at https://login.partner.microsoftonline.cn/. Then select the Admin tile to open the admin center.  
-    
-2. In the left pane, select **Resources** \> **Sites**. (You may need to select **Show all** to see the Resources option.)
-    
-    ![Microsoft 365 admin sites](media/d7757cbe-6531-492f-8547-e055b71d0abf.png)
-  
-3. Select **Add a site**.
-    
-    ![Create a classic site from the Microsoft 365 admin center](media/3026fd12-9d34-4948-a149-fcc8de7b7d83.png)
+1. Sign in to SharePoint Admin Center (Would be at https://[tenant-name]-admin.sharepoint.com) as a SharePoint admin.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/create-site-collection.md:68]:
```suggestion
1. Sign in to the SharePoint admin center (at https://[tenant-name]-admin.sharepoint.com) as a SharePoint admin. 
```

File: SharePoint/SharePointOnline/create-site-collection.md [hunk 2]
```diff
@@ -63,25 +63,27 @@ By using the new SharePoint admin center, you can create sites that use one of t
...
-2. In the left pane, select **Resources** \> **Sites**. (You may need to select **Show all** to see the Resources option.)
-    
-    ![Microsoft 365 admin sites](media/d7757cbe-6531-492f-8547-e055b71d0abf.png)
-  
-3. Select **Add a site**.
-    
-    ![Create a classic site from the Microsoft 365 admin center](media/3026fd12-9d34-4948-a149-fcc8de7b7d83.png)
+1. Sign in to SharePoint Admin Center (Would be at https://[tenant-name]-admin.sharepoint.com) as a SharePoint admin. 
+
+2. In the left pane, select **Sites** \> **Active Sites**.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/create-site-collection.md:70]:
```suggestion
2. In the left pane, select **Sites** > **Active Sites**. 
```

File: SharePoint/SharePointOnline/create-site-collection.md [hunk 3]
```diff
@@ -63,25 +63,27 @@ By using the new SharePoint admin center, you can create sites that use one of t
...
+2. In the left pane, select **Sites** \> **Active Sites**. 
   
-4. Fill out the properties for the site:
+3. Select **Create**.
     
-    ![New Site Collection dialog (top half)](media/1f30a4a6-27b7-42cd-97b0-bcef2e515902.PNG)
+     ![The Create button on the Active sites page](media/create-site-button.png)
+     
   
+4. At the bottom of the panel, click **"Other options"**:
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/create-site-collection.md:77]:
```suggestion
4. At the bottom of the panel, select**Other options**:
```

File: SharePoint/SharePointOnline/create-site-collection.md [hunk 4]
```diff
@@ -63,25 +63,27 @@ By using the new SharePoint admin center, you can create sites that use one of t
...
+
+![image](https://user-images.githubusercontent.com/7239963/112038959-d6da3500-8b19-11eb-8315-7f780ac46649.png)
+
+
+5. From the template drop-down, select "More templates"
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/create-site-collection.md:82]:
```suggestion
5. From the template drop-down, select **More templates**. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**